website/docs/d/pn_application_chrome.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "ibm"
+page_title: "IBM : pn_application_chrome"
+sidebar_current: "docs-ibm-datasource-pn-application-chrome"
+description: |-
+  Get configuration of platform chrome web
+---
+
+# ibm_pn_application_chrome
+
+Provides a read-only data source for platform chrome web. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "pn_application_chrome" "pn_application_chrome" {
+	application_id = "application_id"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `application_id` - (Required, string) Unique ID of the application using the push service.
```

Comment:
Unique ID  change to Unique guid...since id is the crn of the resource

Reply:
done